ansible/roles/compiled/tasks/common/main.yml

```diff
@@ -996,3 +1010,28 @@
   tags:
       - never
       - fftw
+
+- name: Install OpenBLAS
+  vars:
+      versions:
+      - {
+        version_number: "0.3.9",
+        url: "https://github.com/OpenMathLib/OpenBLAS/releases/download/v0.3.9/OpenBLAS-0.3.9.tar.gz",
+        checksum: "md5:28cc19a6acbf636f5aab5f10b9a0dfe1",
+      }
+      - {
+          version_number: "0.3.20",
+          url: "https://github.com/OpenMathLib/OpenBLAS/releases/download/v0.3.20/OpenBLAS-0.3.20.tar.gz",
+          checksum: "md5:abfaa43d995046ca4c56ccf14165c93c",
+      }
+      - {
+        version_number: "0.3.21",
+        url: "0.3.25",
```

Comment:
Oh wait ... i just spotted this. I'm guessing this isn't right.

Reply:
Didn't I already fix that???

Reply:
haha. no problem! that's why a second pair of eyes help!